### PR TITLE
Added a FAQs page.

### PR DESCRIPTION
**Pull request by brettmeehan-google**

### CODE REVIEW

File: faqs.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+identity of the gateway GCP VM. It is used to fetch the image from GCR and run
+the VM itself.
+
+On the *Connector* side: **#2** (Admin User Account) is used when the bash
+script runs *gcloud auth login*. **#2** is then used to fetch the connector
+image for GCR.
+
+Later, the AppLink connector container is started with **#2** (Admin User
+Account) impersonating **#3** (Connector Service Account). Thus, internally all
+IAP TCP IAM bindings are tied to **#3** (Connector Service Account).
```
Comment (anuragbeniwal) [inline faqs.md:117]: s/IAP TCP/Applink related

File: faqs.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+1.  The Default Compute Service Account \
+    (\[*project-id*\]-compute@developer.gserviceaccount.com)
+2.  The Admin User Account
+3.  The Connector Service Account. This is the service account that is created
+    and returned in the [Connector](terraform-config.md#applink-connector) step.
+
+Identities **#1** and **#2** need to be part of the signup request for BCE
+AppLink and provided to your GCP Customer Engineer/Cloud Security Specialist.
+
+On the *Gateway* side: **#1** (Default Compute Service Account) acts as the
```
Comment (anuragbeniwal) [inline faqs.md:107]: s/Gateway/Gateway(GCP)

File: faqs.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+identity of the gateway GCP VM. It is used to fetch the image from GCR and run
+the VM itself.
+
+On the *Connector* side: **#2** (Admin User Account) is used when the bash
```
Comment (anuragbeniwal) [inline faqs.md:111]: s/Connector/Connector(on-prem)

File: faqs.md (hunk 1 above)
Comment (brettmeehan-google) [reply faqs.md]: Done.

File: faqs.md (hunk 2 above)
Comment (brettmeehan-google) [reply faqs.md]: Done.

File: faqs.md (hunk 3 above)
Comment (brettmeehan-google) [reply faqs.md]: Done.

**Review by anuragbeniwal — APPROVED** [review]

**Pull request merged**